scripts/006-pspsdk-stage2.sh

```diff
@@ -5,7 +5,6 @@
  set -e
 
  ## Download the source code if it does not already exist.
- clone_git_repo github.com pspdev pspsdk
```

Comment:
This should probably stay, even if the directory probably already exists. Otherwise we may see some breakage later.